dependencymods.md

```diff
@@ -56,7 +56,15 @@ Relevant Links:
 
 ## [MagicUI](https://github.com/BadMagic100/HollowKnight.MagicUI)
 
-A core mod made by BadMagic.
+A core mod made by BadMagic that simplifies creating non-menu, in-game UI elements. Unity's UI elements are low-level components, meaning you need to be
+very familiar with their implementation details to piece them together into something useful and usually requiring a significant amount of boilerplate code.
+In contrast, MagicUI provides:
+* A hierarchical layout tree composed only of various "arrangable elements," which encapsulate boilerplate code and repeated arrangement behaviors.
```

Comment:
meant to be `"arrangable elements",` ?

Reply:
no, punctuation is always supposed to be inside the quotes. If I were being really picky I'd probably want to do '`ArrangableElement`s' but I don't think the specificity is needed here

Reply:
Looks weird in this context, so i wanted to confirm intent.